vendor/manifest

```diff
@@ -250,6 +250,62 @@
             "branch": "HEAD",
             "notests": true
         },
+        {
+            "importpath": "github.com/ServiceComb/auth",
+            "repository": "https://github.com/ServiceComb/auth",
+            "vcs": "git",
+            "revision": "4cc33395dd156fce9cb4d8ae505f9297601d5442",
```

Comment:
Instead of giving a particular revision can we use the latest tag, because now everytime someone change something in auth then they need to bump this version in go-chassis